README.md

```diff
@@ -85,6 +90,30 @@ study for newcomers.</summary>
 > This models enables to identify materials with deep metric learning applied to X-Ray Diffraction (XRD) spectrum. Read [this post](https://towardsdatascience.com/automatic-spectral-identification-using-deep-metric-learning-with-1d-regnet-and-adacos-8b7fb36f2d5f) for more background.
 </details>
 
+<details>
+<summary><a href="https://github.com/overwindows/SemanticCodeSearch">Semantic Code Search</a> - Retrieving relevant code snippets given a natural language query. <code>NLP</code></summary>
+
+> Different from typical information retrieval tasks, code search requires to bridge the semantic gap between the programming language and natural language, for better describing intrinsic concepts and semantics. The repository provides the pretrained models and source code for [Learning Deep Semantic Model for Code Search using CodeSearchNet Corpus](https://arxiv.org/abs/2201.11313), where they apply several tricks to achieve this.
+</details>
+
+<details>
+<summary><a href="https://git.tu-berlin.de/rsim/duch">DUCH: Deep Unsupervised Contrastive Hashing</a> - Large-scale cross-modal text-image retrieval in remote sensing with computer vision. <code>CV</code> <code>NLP</code></summary>
+
+</details>
+
+<details>
+<summary><a href="https://github.com/geekinglcq/HRec">DUration: Deep Unsupervised Representation for Heterogeneous Recommendation</a> - Recommending different types of items efficiently. <code>RecSys</code></summary>
+
+> State-of-the-art methods are incapable of leveraging attributes from different types of items and thus suffer from data sparsity problems because it is quite challenging to represent items with different feature spaces jointly. To tackle this problem, they propose a kernel-based neural network, namely deep unified representation (DURation) for heterogeneous recommendation, to jointly model unified representations of heterogeneous items while preserving their original feature space topology structures. See [paper](https://arxiv.org/abs/2201.05861).
+</details>
+
+<details>
+<summary><a href="https://github.com/selimfirat/addml">ADDML</a> - Unsupervised anomaly detection via deep metric learning</summary>
+
+> Source code for [Unsupervised Anomaly Detection via Deep Metric Learning with End-to-End Optimization](https://arxiv.org/abs/2005.05865), where they present a novel data distillation method through self-supervision to remedy the conventional practice of assuming all data as normal, employing the hard mining technique from the DML literature. They show these components improve the performance of the model, significantly reducing the running time.
+</details>
```

Comment:
I did not understand this  one. Maybe if you could dumb it down for me, I could write some more intuitive short description. Also code does not look awesome.